dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -452,31 +481,31 @@ else if(f.getFieldType().equals("time")) {
                 		m.put(st.getVelocityVarName() + "." + f.getVelocityVarName(),"");
                 	}
                 }
-                else if (f.getFieldType().equals("date")) {
+                else if (f.getFieldType().equals(ESMappingConstants.FIELD_ELASTIC_TYPE_DATE)) {
                     try {
                         String dateString = dateFormat.format(valueObj);
                         m.put(st.getVelocityVarName() + "." + f.getVelocityVarName(), dateString);
                     }
                     catch(Exception ex) {
                         m.put(st.getVelocityVarName() + "." + f.getVelocityVarName(),"");
                     }
-                } else if(f.getFieldType().equals("date_time")) {
+                } else if(f.getFieldType().equals(ESMappingConstants.FIELD_TYPE_DATE_TIME)) {
                     try {
                         String datetimeString = datetimeFormat.format(valueObj);
                         m.put(st.getVelocityVarName() + "." + f.getVelocityVarName(), datetimeString);
                     }
                     catch(Exception ex) {
                         m.put(st.getVelocityVarName() + "." + f.getVelocityVarName(),"");
                     }
-                } else if (f.getFieldType().equals("category")) {
+                } else if (f.getFieldType().equals(ESMappingConstants.FIELD_TYPE_CATEGORY)) {
                     // moved the logic to loadCategories
-                } else if (f.getFieldType().equals("checkbox") || f.getFieldType().equals("multi_select")) {
-                    if (f.getFieldContentlet().startsWith("bool")) {
+                } else if (f.getFieldType().equals(ESMappingConstants.FIELD_TYPE_CHECKBOX) || f.getFieldType().equals(ESMappingConstants.FIELD_TYPE_MULTI_SELECT)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 164 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)